runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/internal/registry/RegisterTask.java

```diff
@@ -307,9 +270,12 @@ else if ("http".equals(configURL.getProtocol()) || "https".equals(configURL.getP
         {
             errors.forEach(msg -> errorHandler.onError(new JsonException(msg)));
         }
-
-        // TODO: repeat to detect and apply changes
-
         return null;
     }
+
+    public void setConfigText(
+        String configText)
+    {
+        this.configText = configText;
+    }
```

Comment:
This state field seems necessary only to temporarily hold a parameter for RegisterTask prior to submitting on a different thread.

This can be removed after changing to a direct method call on RegisterTask, passing as a parameter instead.

Note: RegisterTask should probably be renamed as it's not really a task.